functions/v2/pubsub/subscribe-to-topic/src/main/java/functions/SubscribeToTopic.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package functions.v2;
+
+// [START functions_cloudevent_pubsub]
+
+import com.example.cloud.functions.v2.eventpojos.PubSubBody;
+import com.google.cloud.functions.CloudEventsFunction;
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import io.cloudevents.CloudEvent;
+import java.nio.charset.StandardCharsets;
+import java.util.Base64;
+import java.util.logging.Logger;
+
+public class SubscribeToTopic implements CloudEventsFunction {
+  private static final Logger logger = Logger.getLogger(SubscribeToTopic.class.getName());
+
+  @Override
+  public void accept(CloudEvent event) {
+    if (event.getData() == null) {
```

Comment:
Does this happen on GCF V2? If not, it's probably extraneous (...unless it's required by the compiler - Java itself can be nitpicky!)

Reply:
I've removed this check - it was leftover from the previous sample version :)